pkg/deploymanager/common.go

```diff
@@ -0,0 +1,27 @@
+package deploymanager
+
+import (
+	"context"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CreateNamespace creates a namespace in the cluster, ignoring if it already exists
```

Comment:
Function level comments should end in a period(.)
(hint: see if `godot` helps?)

Reply:
Corrected